src/main/java/com/powsybl/openloadflow/ac/nr/PerEquationTypeStoppingCriteria.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.ac.nr;
+
+import com.powsybl.openloadflow.ac.equations.AcEquationType;
+import com.powsybl.openloadflow.ac.equations.AcVariableType;
+import com.powsybl.openloadflow.equations.EquationSystem;
+import com.powsybl.openloadflow.equations.Vectors;
+import com.powsybl.openloadflow.util.PerUnit;
+
+/**
+ * @author Alexandre Le Jean <alexandre.le-jean at artelys.com>
+ */
+public class PerEquationTypeStoppingCriteria implements NewtonRaphsonStoppingCriteria {
+
+    // FIXME Any idea for the name?
+    private final double defaultValueAlpha = Math.pow(10, -5);
+
+    // FIXME Any idea for the name?
+    private final double defaultValueRho = Math.pow(10, -5);
+
+    // FIXME Any idea for the name?
+    private final double defaultValueB = Math.pow(10, -5);
```

Comment:
maxDefaultSusceptanceMismatch.

Maybe Math.pow(10, -5) is too big. I do not well know typical values for B susceptance of shunts. Maybe Math.pow(10, -7) is more secure.